packages/dso-toolkit/src/components/table/table.scss

```diff
@@ -96,6 +96,69 @@ table.table,
     }
   }
 
+  &.dso-ozon-table {
+    &.dso-table-outside-lines-top {
+      border-top: 1px solid table.$th-border-color;
```

Comment:
logical properties.

Geldt voor alle `border`-declaraties in dit document (nog 7 hieronder)

Reply:
Wat is de policy? Direct alle `border`-declaraties omzetten naar logical properties, dus ook die ik niet geraakt heb in de PR?

Reply:
doe alleen maar je eigen declaraties. het is eigenlijk alleen een kwestie van 'keeping the campground clean'. er staat voor mij nog een mega-operatie op stapel om alle physicals naar logicals om te katten.